framework/integration/client/nodeconfig/client.go

```diff
@@ -0,0 +1,165 @@
+// +build k8srequired
+
+package nodeconfig
+
+import (
+	"time"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/e2e-harness/pkg/harness"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/operatorkit/client/k8scrdclient"
+	"github.com/giantswarm/operatorkit/framework"
+	"github.com/giantswarm/operatorkit/framework/integration/client"
+	"github.com/giantswarm/operatorkit/framework/integration/testresourceset"
+	"github.com/giantswarm/operatorkit/informer"
+)
+
+type Client struct {
```

Comment:
I am not happy with my naming here at all, I couldn't think of anything better. So any suggestions are welcome.

Reply:
Uncertain. Wrapper could work for me.

Reply:
I renamed this to `Wrapper` everywhere, not sure if this is better. I am fine with this.